### PR TITLE
681 unify agent endpoints

### PR DESCRIPTION
**Pull request by satoshisound**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,9 @@ but that should be it.
 
 Only thing that is mandatory is apikey which can also be provided through `API_KEY` environment variable.
 
-By default it will try to communicate with LND using local gRPC connection and `readonly.macaroon`.
+To obtain an API key, you will need a <a href="https://bolt.observer">bolt.observer</a> account.
```
Comment (fiksn) [inline README.md:54]: In markdown links are ala:
```
[bolt.observer](https://bolt.observer)
```

**Review by fiksn — COMMENTED** [review]
What is missing is the usage of the new `nodedata` in `balance-agent` cmd line utility.

**Comment by satoshisound** [comment]
> 

For adding nodedata to balance-agent, am I replacing the old checker from channelchecker or adding a checker for nodedata alongside what you have there already?

**Comment by fiksn** [comment]
> @fiksn
> 
> For adding nodedata to balance-agent, am I replacing the old checker from channelchecker or adding a checker for nodedata alongside what you have there already?

Yeah, `nodeinfo` and `channelchecker` can be removed in favor of that new `nodedata`. But we can't release a new agent version until we have support for that on API side in production.

File: cmd/balance-agent/main.go [hunk 2]
```diff
@@ -52,8 +50,7 @@ var (
 	url                 string
 	nodeurl             string
 	// GitRevision is set with build
-	GitRevision = "unknownVersion"
-
+	GitRevision      = "unknownVersion"
 	nodeInfoReported sync.Map
```
Comment (fiksn) [inline cmd/balance-agent/main.go:54]: This is also obsolete, was used as a mutex to not report balance before reporting nodeinfo.

**Comment by fiksn** [comment]
Another thing is that `channelchecker` and `nodeinfo` should get removed finally in favor of your `nodedata` abstraction.

File: cmd/balance-agent/main.go (hunk 2 above)
Comment (fiksn) [reply cmd/balance-agent/main.go]: We need `GitRevision`, removed `nodeInfoReported`

**Review by tirithtech — APPROVED** [review]

**Pull request merged**